src/commonMain/kotlin/fr/acinq/lightning/json/JsonSerializers.kt

```diff
@@ -512,6 +514,9 @@ object JsonSerializers {
     @Serializer(forClass = CommitSigTlv.AlternativeFeerateSigs::class)
     object CommitSigTlvAlternativeFeerateSigsSerializer
 
+    @Serializer(forClass = CommitSigTlv.Batch::class)
+    object CommitSigTlvBatchSerializer
```

Comment:
What prompted you to add this serializer? Was something failing?

Reply:
Yes, tests are otherwise failing.